oshc/main/static/main/css/app.css

```diff
@@ -12,6 +12,7 @@
 
 body {
   overflow-x: hidden;
+  padding-top: 50px;
```

Comment:
It's generally not a good practice to give any kind of padding or margin to the body, it certainly affects the whole Html structure.

I guess you were trying to give padding to `.main-section`. Do the same and it will work.